csharp/src/Drivers/Apache/Thrift/Service/Rpc/Thrift/TBinaryColumn.cs

```diff
@@ -83,14 +84,14 @@ public TBinaryColumn DeepCopy()
 
                   values = new ArrowBuffer.Builder<byte>();
                   int offset = 0;
-                  offsetBuffer = new byte[(length + 1) * 4];
+                  offsetBuffer = new byte[(length + 1) * IntSize];
                   var memory = offsetBuffer.AsMemory();
                   var typedMemory = Unsafe.As<Memory<byte>, Memory<int>>(ref memory).Slice(0, length + 1);
```

Comment:
We don't seem to be using `typedMemory` any more. Applies to some of the other column types as well.

Reply:
Removed reference to `typedMemory`.